models/staff.py

```diff
@@ -13,15 +14,21 @@ def __init__(self, first_name, last_name):
         self.person_type = 'STAFF'
         self.staff_id = self.get_staff_id()
         Staff.staff_data.append(self.staff_id)
+        self.office_allocated = False
+        self.office = ''
 
     def get_staff_id(self):
         """Returns unique id for room by generating random string"""
-        id = 'S' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(7))
+        staff_id = 'S' + \
+            ''.join(random.choice(string.ascii_uppercase + string.digits)
+                    for _ in range(7))
 
         """Checks if the random id created already exists in the array"""
-        while id in Staff.staff_data:
+        while staff_id in Staff.staff_data:
```

Comment:
Because i declared outside the __init__ statement but since it is also a class variable, i'll use self.staff_data.
Thanks